docs/development.md

```diff
@@ -504,3 +504,50 @@ Issue is a duplicate of other already existing issue.
 #### [invalid](https://github.com/kedgeproject/kedge/labels/invalid)
 
 #### [wontfix](https://github.com/kedgeproject/kedge/labels/wontfix)
+
+
+## Code instructions
```

Comment:
I would move it up, before the Labeling guide section

Reply:
done!